Content.Shared/Psionics/PsionicPowerPrototype.cs

```diff
@@ -36,10 +37,37 @@ public sealed partial class PsionicPowerPrototype : IPrototype
     public ComponentRegistry Components = new();
 
     /// <summary>
-    ///     What message will play as a popup when the power is initialized.
+    ///     What message will be sent to the player as a Popup.
+    ///     If left blank, it will default to the Const "generic-power-initialization-feedback"
     /// </summary>
-    [DataField(required: true)]
-    public string InitializationFeedback = "psionic-power-initialization-default";
+    [DataField]
+    public string? InitializationPopup;
+
+    /// <summary>
+    ///     What message will be sent to the chat window when the power is initialized. Leave it blank to send no message.
+    ///     Initialization messages won't play for powers that are Innate, only powers obtained during the round.
+    ///     These should generally also be written in the first person, and can be far lengthier than popups.
+    /// </summary>
+    [DataField]
+    public string? InitializationFeedback;
+
+    /// <summary>
+    ///     What color will the initialization feedback display in the chat window with.
+    /// </summary>
+    [DataField]
+    public string InitializationFeedbackColor = "#8A00C2";
```

Comment:
Make this something that contrasts better with chat

Reply:
![image](https://github.com/user-attachments/assets/1af05ee5-967c-4377-857c-cc64352da382)

I'll take it one more step up the chain. That should be good. :)